src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporter.cs

```diff
@@ -77,6 +81,13 @@ public OtlpTraceExporter(OtlpExporterOptions options)
         {
             this.exportClient = exporterOptions.GetTraceExportClient();
         }
+
+        transmissionHandler ??= new OtlpExporterTransmissionHandler<OtlpCollector.ExportTraceServiceRequest>();
+
+        transmissionHandler.ExportClient = this.exportClient;
+        transmissionHandler.Options = exporterOptions;
```

Comment:
I assume new fields for users to specify the retry behavior, such as the FileBlobProvider, and timeout will be introduced in `OtlpExporterOptions` in the follow-up PRs?

Reply:
No, we will not introduce any new public options. The retry feature will be introduced as an experimental feature. Configuration options will be provided via feature flags(env variables).